map_data/geographical_region.txt

```diff
@@ -794,25 +822,104 @@ custom_eastern_kingdoms_jungles = {
 }
 custom_jungles_except_pandaria = {
 	regions = {
-		world_northrend_sholazar_basin
 		custom_eastern_kingdoms_jungles
 		custom_south_seas_except_deserts
+		world_northrend_sholazar_basin
 	}
 }
 custom_jungles = {
 	regions = {
 		custom_jungles_except_pandaria
+		world_kalimdor_feralas
 		world_pandaria_jade_forest
 		world_pandaria_krasarang_wilds
 	}
+	duchies = {
+		d_echo_isles
+		d_seething_shore
+		d_glaive_isle
+       		d_lost_isles
+	}
```

Comment:
Move these regions and duchies to `custom_jungles_except_pandaria`. Feralas indeed isn't in Pandaria.